server.cjs

```diff
@@ -10,16 +10,20 @@ app.use(express.static(path.join(__dirname, '/src')))
 app.use(express.static(__dirname))
 app.use(express.json())
 app.get('/all-city-timezones', function (req, res) {
-  allCityData = timezone.allTimeZones()
-  res.json(allCityData)
+  const childProcess = fork('./childThread.cjs')
```

Comment:
The forked process would remain open after the response is obtained. Can you try closing/killing the childProcess on getting back the response?